default_lcf.xml

```diff
@@ -56,16 +56,20 @@
 
         <!-- R File Appender  set to output to a single log file.
             This is defined for systems that can't (or don't want to) have rolling files. -->
-        <File name="R" fileName="${sys:jmri.log.path}session.log" append="false">
+        <RollingFile name="R" fileName="${sys:jmri.log.path}session.log"
+            filePattern="${sys:jmri.log.path}session.log" append="false">
             <PatternLayout pattern="%d{ISO8601} %-37.37c{2} %-5p - %m [%t]%n"/>
-        </File>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="5MB"/>
+            </Policies>
+        </RollingFile>
 
         <!-- T Rolling File Appender is set to output to a rolling file.
             This only works for certain systems, but manages files in a convenient way.
             T is defined to preserve messages between sessions and to keep up to
             2 previous log files in addition to the current. -->
         <RollingFile name="T" fileName="${sys:jmri.log.path}messages.log"
-            filePattern="${sys:jmri.log.path}messages.log.%i" append="true">
+            filePattern="${sys:jmri.log.path}messages.%i.log" append="true">
```

Comment:
I think it would be good to mention this in the release notes.

Reply:
Indeed, updated.